_sass/_old_vanilla.scss

```diff
@@ -0,0 +1,4805 @@
+.p-icon--minus, .p-icon--plus, .p-icon--expand, .p-icon--collapse, .p-icon--chevron, .p-icon--close, .p-icon--help, .p-icon--information, .p-icon--delete, .p-icon--error, .p-icon--warning, .p-icon--external-link, .p-icon--contextual-menu, .p-icon--code, .p-icon--menu, .p-icon--copy, .p-icon--search, .p-icon--success, .p-icon--share, .p-icon--user, .p-icon--question, .p-icon--spinner {
```

Comment:
Why do you need `old_vanilla`? I thought both the docs.vanillaframework.io styling and this website were based off vanilla-framework 1.8.0? Or at least that they'd be close enough to work...?

Reply:
This is for the sidebar. 
I needed the `.p-sidebar` class. 
@anthonydillon is there a better way to do this? Or wait until sth lands in Vanilla and then come back here and fix?

Reply:
You should be able to port over just that class. No need to bring the whole file over.

Reply:
Oh no we should use the styling in the existing docs pages. For example: https://github.com/canonical-websites/www.ubuntu.com/blob/master/static/sass/_layout_documentation.scss

Reply:
Thanks, looking good. Just need to add:
```css
.docs-container {
  max-width: 64.875rem;
  margin: 0 auto;
}
```